scripts/untranslated.xml

```diff
@@ -6,4 +6,6 @@
     <string name ="webxdc_empty_hint">Received or sent apps will appear here. Tap \"Files\" to select downloaded apps.</string>
     <string name ="shortcut_share_btn">Click the share button</string>
     <string name ="shortcut_add_to_home_description">Select - Add to Home Screen - to add the app to your home screen.</string>
+    <string name="connectivity_low_data_mode">Low Data Mode: background message fetching is disabled.</string>
+    <string name="connectivity_low_battery_mode">Low Power Mode: background message fetching is disabled.</string>
```

Comment:
```suggestion
    <string name="connectivity_low_power_mode">Disabled by system's \"Low Power Mode\"".</string>
```

i'd name the string correctly here, and adapt translation similar to the other strings.

according to https://support.apple.com/en-us/HT205234 - things are "reduced or affected" - so, maybe we can also say "Delayed by system's \"Low Power Mode\"", however, i have the feeling it is always close to "Disabled", so maybe better saying that, even if it is too pessimistic in some cases.